### DISCUSSION TITLE
Bana burayı anlatacak birisi var mı

### DISCUSSION THREAD

**Discussion by kkcemrecan** [IBAX-io/go-ibax#877] [Q&A] [answered]
### Clear and concise description of the problem

Arkadaşlar yeni katıldım bana öğretecek var mı

### Suggested solution

Arkadaşlar yeni katıldım bana öğretecek var mı

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by furkanzal** [comment]
Good project

**Comment by Azizatli21** [comment]
Bende çözemedim

**Comment by Efsane4167** [comment]
> ### Sorunun açık ve özlü açıklaması
> Arkadaşlar yeni katıldım bana öğretecek var mı
> 
> ### Önerilen çözüm
> Arkadaşlar yeni katıldım bana öğretecek var mı
> 
> ### Alternatif
> _Cevap yok_
> 
> ### Ek bağlam
> _Cevap yok_
> 
> ### doğrulamalar
> * [x] [Belgeleri](https://docs.ibax.io) okuyun .
> * [x] Kopya oluşturmaktan kaçınmak için aynı özelliği talep eden bir sorun olup olmadığını kontrol edin.

**Comment by Efsane4167** [comment]
Suan bisi anlamis degilim bende cok karmasik.githup ne alaka burdan yorum yap taetisma konusu acarak puan kazanmak ne alaka cozemedim. Telegram grubu git ordan ogren

**Comment by cryptopus58** [comment]
Telegram grubundan bilgi alabilirsiniz

**Comment by Grkmkgl** [comment]
Ordada bisey yok

**Comment by ibaxio** [comment, marked as answer]
Join Telegram group, learning from documentation (https://docs.ibax.io/), get more in official website (https://ibax.io), and try to write an application.

Github Issues (https://github.com/IBAX-io/go-ibax/issues) is only for developers, please post your thread in Discussion (https://github.com/IBAX-io/go-ibax/discussions) .

**Comment by hsynkader** [comment]
Github daha çok geliştiriciler için, düşük seviyede olsa kod bilginiz olması lazım. Onun dışında yorum yapmaktan fazla yapacak bir şey yok.
